fractions/javaee/bean-validation/README.adoc

```diff
@@ -0,0 +1,4 @@
+# Bean Validation
+
+The Bean Validation fraction provides support for JSR 303,
```

Comment:
In the spirit of minimalism, where we omit words that do not add value to the message, I suggest removing _The Bean Validation fraction_ and starting the sentence with _Provides support..._ because no meaning is lost then. This is valid in all fraction READMEs that contain this wording.